src/Sulu/Bundle/SecurityBundle/Resources/js/containers/RoleAssignments/RoleAssignment.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import React from 'react';
+import {observer} from 'mobx-react';
+import {MultiSelect} from 'sulu-admin-bundle/components';
+import type {Localization} from 'sulu-admin-bundle/stores';
+import roleAssignmentsStyle from './roleAssignments.scss';
+
+type Props = {
+    localizations: Array<Localization>,
+    onChange: (value: Object) => void,
+    value: Object,
+};
+
+@observer
+export default class RoleAssignments extends React.Component<Props> {
+    handleChange = (newLocalizations: Array<string>) => {
+        const newValue = {...this.props.value};
+        newValue.locales = newLocalizations;
+
+        this.props.onChange(newValue);
+    };
+
+    render() {
+        const {localizations, value} = this.props;
+
+        return (
+            <div className={roleAssignmentsStyle.roleAssignmentContainer}>
```

Comment:
Why don't we use an actual `table` here? Then the size of the columns would be spread with a better algorithm.

Reply:
Let's about this next week with @alexander-schranz